@xen-orchestra/web-core/lib/components/ui/input/UiTextarea.vue

```diff
@@ -0,0 +1,111 @@
+<!-- v2 -->
+<template>
+  <div class="ui-textarea">
+    <UiLabel v-if="slots.default" :accent="labelAccent" :required :icon :href><slot /></UiLabel>
+    <textarea v-model="model" :class="accent" :disabled class="textarea" v-bind="$attrs" />
```

Comment:
It is not documented yet, but in the new guidelines, we should avoid using `$attrs` in the template (same as for the `$slots`), so it would be:
```html
<template>
    <textarea v-model="model" :class="accent" :disabled class="textarea" v-bind="attrs" />
</template>

<script lang="ts" setup>
// ...

const slots = defineSlots<{...}>() // Just to show the place where 'useAttrs()' would go

const attrs = useAttrs()

// ...
</script>
```